api/src/main/java/io/grpc/Server.java

```diff
@@ -104,8 +104,8 @@ public int getPort() {
 
   /**
    * Initiates an orderly shutdown in which preexisting calls continue but new calls are rejected.
```

Comment:
I think "initiates" is the key word here.

I think this would probably go better as a second paragraph that is merely informative. Something like:

> Note that this method does not wait for the server to stop, so existing calls may continue after returning. To stop existing calls use {`@link` #shutdownNow()} and to wait for the server to fully stop use {`@link` #awaitTermination()}.

Reply:
Combined your and @sanjaypujare's suggestions: https://github.com/grpc/grpc-java/pull/6512/commits/af9a157931771751dacddc334f578c557e6afaac